fix(sdk): changed keys to keysOrQuery to accept both keys and query in updateItems (#20774)
directus/directus: .changeset/heavy-meals-poke.md

---

> '@directus/sdk': patch

Reviewer: That's a breaking change, right @br41nslug?
```suggestion
'@directus/sdk': major
```

Author: i dont think its a breaking change, it is backward compatible ryt?

Reviewer: Ah right, `keys` still works 👍 

We might make it a minor then, if we treat it as a new feature (@br41nslug 👀)
```suggestion
'@directus/sdk': minor
```

Reviewer 2: I personally consider it to just another bug fix so `patch` seems fine to me. `minor` seems fine to as improvement to be better in sync with the API but a `major` it is surely not.

Reviewer: Yeah, we can certainly consider it a patch too 👍

---

> make either keys or query required in updateItems

Reviewer: ```suggestion
Made either keys or query argument mandatory for `updateItems`
```

> changed keys to keysOrQuery to accept both keys and query in updateItems

Reviewer: ```suggestion
Changed `keys` argument in `updateItems` method to `keysOrQuery` to accept both formats
```
